pkg/podman/listener.go

```diff
@@ -42,3 +49,23 @@ func preparePodmanCommand(out io.Writer) *exec.Cmd {
 
 	return cmd
 }
+
+func waitForPodmanSock() error {
+	const (
+		retries      = 5
+		sleepSeconds = 3
+	)
+
+	zap.S().Infof("Waiting for '%s' to be created", podmanSocketPath)
+	for i := 0; i < retries; i++ {
+		if _, err := os.Stat(podmanSocketPath); err == nil {
+			zap.S().Infof("'%s' file has been created successfully", podmanSocketPath)
+			return nil
+		}
+
+		zap.S().Infof("'%s' file is not yet created, retrying in %d seconds", podmanSocketPath, sleepSeconds)
+		time.Sleep(sleepSeconds * time.Second)
```

Comment:
What happens if I ctrl+C during a sleep? Given the short sleep it's probably not an issue, I just want to double check that we're not going to lock the user out of exiting.

Reply:
ctrl+C breaks the sleep, so we wouldn't block the user out of exiting.